index.html

```diff
@@ -1,431 +1,247 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
```

Comment:
Please use linter config for html+css+js from [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js). Moreover, this branch has merge conflicts. Kindly remove them.